src/views/Transformer.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <main>
+    <ViewHeader :title="transformer.name" :backLink="{ link: { name: 'transformers' } }"/>
+
+    <ListSection class="preview-container" :list="transformerList"/>
+
+    <div class="row">
+      <ListSection title="Attributes" :list="transformerAttributesList"/>
+      <ListSection title="Actions" :list="transformerActionsList"/>
+    </div>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import {
+  Transformer,
+  Transformers,
+  createTransformerNullObject,
+} from '@/apis/rembrandt/rembrandt';
+import ViewHeader from '@/components/ViewHeader.vue';
+import ListSection from '@/components/ListSection.vue';
+import { ListEntry } from '@/components/Li.vue';
+import Utils from '@/utils/Utils';
+import { NotificationLevel } from '@/interfaces/Notification';
+
+@Component({
+  components: {
+    ListSection,
+    ViewHeader,
+  },
+})
+export default class Algorithm extends Vue {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  public transformer: Transformer;
+
+  public get transformerList(): ListEntry[] {
+    return Utils.transformerToList([ this.transformer ]);
+  }
+
+
+  public get transformerAttributesList(): ListEntry[] {
+    return Utils.transformerAttributesToList(this.transformer);
+  }
+
+  public get transformerActionsList(): ListEntry[] {
+    const transformerActions = [];
+
+    transformerActions.push({
```

Comment:
We could think about adding a create button here too for convenience.

Reply:
yeah, sure. I will do this.

Reply:
done!